Tribler/community/multichain/community.py

```diff
@@ -19,11 +19,6 @@
 from Tribler.dispersy.message import Message, DelayPacketByMissingMember
 from Tribler.dispersy.resolution import PublicResolution
 
-SIGNATURE = u"signature"
```

Comment:
These lines were removed form the `devel` branch while this pull request was being developed. These changes are merely made to stay up to date with the default branch.